usaspending_api/etl/es_etl_helpers.py

```diff
@@ -290,78 +306,91 @@ def es_data_loader(client, fetch_jobs, done_jobs, config):
     return
 
 
-def streaming_post_to_es(client, chunk, index_name, job_id=None, doc_type="transaction_mapping"):
+def streaming_post_to_es(client, chunk, index_name, job_id=None):
     success, failed = 0, 0
     try:
-        for ok, item in helpers.streaming_bulk(client, chunk, index=index_name, doc_type=doc_type):
+        # "doc_type" is set in the index templete file. Don't change this without changing in json file first
+        for ok, item in helpers.streaming_bulk(client, chunk, index=index_name, doc_type="transaction_mapping"):
             success = [success, success + 1][ok]
             failed = [failed + 1, failed][ok]
 
     except Exception as e:
-        print("MASSIVE FAIL!!!\n\n{}\n\n{}".format(str(e)[:5000], "*" * 80))
+        print("Fatal error: \n\n{}...\n\n{}".format(str(e)[:5000], "*" * 80))
         raise SystemExit(1)
 
     printf({"msg": "Success: {}, Fails: {}".format(success, failed), "job": job_id, "f": "ES Ingest"})
     return success, failed
 
 
-def put_alias(client, index, alias_name, award_type_codes):
-    alias_body = {"filter": {"terms": {"type": award_type_codes}}}
+def put_alias(client, index, alias_name, alias_body):
     client.indices.put_alias(index, alias_name, body=alias_body)
 
 
 def create_aliases(client, index, silent=False):
     for award_type, award_type_codes in INDEX_ALIASES_TO_AWARD_TYPES.items():
-        alias_name = "{}-{}".format(settings.TRANSACTIONS_INDEX_ROOT, award_type)
+        alias_name = "{}-{}".format(settings.ES_TRANSACTIONS_READ_ALIAS_PREFIX, award_type)
         if silent is False:
             printf(
                 {
-                    "msg": 'Putting alias "{}" with award codes {}'.format(alias_name, award_type_codes),
-                    "job": "",
+                    "msg": "Putting alias '{}' on {} with award codes {}".format(alias_name, index, award_type_codes),
+                    "job": None,
                     "f": "ES Alias Put",
                 }
             )
-        put_alias(client, index, alias_name, award_type_codes)
+        alias_body = {"filter": {"terms": {"type": award_type_codes}}}
+        put_alias(client, index, alias_name, alias_body)
+
+    # ensure the new index is added to the alias used for incremental loads.
+    # If the alias is on multiple indexes, the loads will fail!
+    printf(
```

Comment:
would a logger be inappropriate here?

Reply:
Not inappropriate. This was written before the logger was standardized in the app. There is no need to change the current function now and it works very well for the script.